packages/trading/src/tradingSdk.ts

```diff
@@ -23,8 +23,6 @@ export type WithPartialTraderParams<T> = T & Partial<TraderParameters>
 export interface TradingSdkOptions {
   enableLogging: boolean
   orderBookApi: OrderBookApi
-  utmContent?: string
-  disableUtm?: boolean
```

Comment:
Not used?

Reply:
Nope, this was an artifact of deleted code